src/xrGame/ui/UIGameTutorial.cpp

```diff
@@ -118,6 +119,7 @@ bool CUISequencer::Start(LPCSTR tutor_name)
 
     Device.seqFrame.Add(this, REG_PRIORITY_LOW - 10000);
 
+    m_name = tutor_name;
```

Comment:
And we also should set it to the default name when tutorial ends\stops.

Reply:
Updated

Reply:
Did you mean set to nullptr when tutorial ends\stops?

Reply:
@ChugunovRoman, yes

Reply:
Fixed, but I'm not sure...